fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -401,7 +401,7 @@ public LogicalPlan visitCTE(CTERelation node, ExpressionMapping context) {
 
             return new LogicalPlan(
                     new OptExprBuilder(new LogicalCTEConsumeOperator(node.getCteId(), cteOutputColumnRefMap),
-                            Collections.emptyList(), new ExpressionMapping(expressionMapping.getScope(), cteOutputs)),
+                            Collections.emptyList(), new ExpressionMapping(node.getScope(), cteOutputs)),
```

Comment:
What's the different from `node.getScope()` and `expressionMapping.getScope()`?

Reply:
Added details in the PR description